src/RFC-0312_DANHighLevelSpecification.md

```diff
@@ -172,6 +172,16 @@ of the contract.
 The asset issuer will also draft and publish the [contract constitution]. The constitution defines _how_ a contract is
 run, and defines the conditions under which the terms of the constitution can be changed.
 
+### The Constitution Committee
+[constitution committee]: #constitution_committee
+[CC]: #cc
+
+In the [asset issuer] will in the creation of the [contract constitution] nominate a key or set of keys to "own" the asset
```

Comment:
Something is wrong with the grammar here

Reply:
thanks fixed